R/stack.R

```diff
@@ -507,6 +509,14 @@ getVarInEnv <- function(name, env) {
   })
 }
 
+getActiveBinding <- function(name, env){
+  ret <- list(
+    bindingFunction = activeBindingFunction(name, env)
```

Comment:
`activeBindingFunction` only exists when R >= 4.0.0. We might need to backport it so that it works with R < 4.0.0.

One simple we to get the binding function that works with all R versions (to my knowledge) is `as.list.environment(env)[[name]]`. We might want to use this for R < 4.0.0?

Reply:
Thanks. I currently only have 4.0.0 installed on my machine, but there this seems to work fine.

Reply:
There is one serious problem with `as.list.environment`: it forces evaluation. 

Consider:
```r
e <- new.env()
delayedAssign("x", stop("do not evaluate 'x'"), assign.env = e)
as.list.environment(e)
# Error in as.list.environment(e) : do not evaluate 'x'
```

Reply:
Uh you're right. Thanks! (https://github.com/ManuelHentschel/vscDebugger/pull/51)

Reply:
Good catch!